src/cogent3/app/io_new.py

```diff
@@ -14,28 +45,263 @@
 __status__ = "Alpha"
 
 
+class tabular(Enum):
+    table = "table"
+    distances = "distances"
+    motif_counts = "motif_counts"
+    motif_freqs = "motif_freqs"
+    pssm = "pssm"
+
+
+def _read_it(path):
+    try:
+        data = path.read()
+    except AttributeError:
+        try:
+            data = path.read_text()
+        except AttributeError:
+            raise IOError(f"unexpected type {type(path)}")
+    return data
+
+
+def _load_seqs(path, klass, parser, moltype):
+    data = _read_it(path)
+    data = data.splitlines()
+    data = dict(iter(parser(data)))
+    seqs = klass(data=data, moltype=moltype)
+    seqs.info.source = str(path)
+    return seqs
+
+
+@define_app(app_type=LOADER)
+class load_aligned:
```

Comment:
As you know, all classes have the same name as io.py. This may cause problems for users.

Reply:
Those are distinguished by name space, and I have an idea how to change how users get app instances that will make the impact on users smaller.